[BUG] Ubuntu dependency installation error with 1.20.0 release

**Context:**
- Playwright Version: 1.20.0
- Operating System: Mac
- Python version: 3.8
- Browser: All

`playwright install-deps`

**Describe the bug**
Error occurred in CircleCI installation of playwright dependencies. Pinning version back to 1.19.1 fixed the issue. This was the specific output from circleCI:

Installing Ubuntu dependencies...
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Hit:2 http://deb.debian.org/debian buster InRelease  
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Get:4 http://deb.debian.org/debian buster-updates/main amd64 Packages.diff/Index [10.1 kB]
Get:5 http://deb.debian.org/debian buster-updates/main amd64 Packages 2022-01-13-2008.29.pdiff [566 B]
Get:6 http://deb.debian.org/debian buster-updates/main amd64 Packages 2022-01-24-2024.03.pdiff [1124 B]
Get:6 http://deb.debian.org/debian buster-updates/main amd64 Packages 2022-01-24-2024.03.pdiff [1124 B]
Get:7 http://security.debian.org/debian-security buster/updates/main amd64 Packages [316 kB]
Fetched 445 kB in 0s (1042 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'libfontconfig1' instead of 'libfontconfig'
Package ttf-ubuntu-font-family is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package libjpeg-turbo8 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'ttf-ubuntu-font-family' has no installation candidate
E: Unable to locate package libicu66
E: Package 'libjpeg-turbo8' has no installation candidate
E: Unable to locate package libvpx6
Failed to install browser dependencies
Error: Installation process exited with code: 100
make: *** [Makefile:85: install] Error 1

--- a/packages/playwright-core/src/utils/dependencies.ts
+++ b/packages/playwright-core/src/utils/dependencies.ts
@@ -21,6 +21,7 @@ import childProcess from 'child_process';
 import * as utils from './utils';
 import { buildPlaywrightCLICommand } from './registry';
 import { deps } from './nativeDeps';
+import { getUbuntuVersion } from './ubuntuVersion';
 
 const BIN_DIRECTORY = path.join(__dirname, '..', '..', 'bin');
 
@@ -53,6 +54,8 @@ export async function installDependenciesWindows(targets: Set<DependencyGroup>, 
 }
 
 export async function installDependenciesLinux(targets: Set<DependencyGroup>, dryRun: boolean) {
+  if (await getUbuntuVersion() === '')
+    throw new Error(`Unsupported Linux distribution, only Ubuntu is supported!`);
   const libraries: string[] = [];
   for (const target of targets) {
     const info = deps[utils.hostPlatform];

